MdeModulePkg/Universal/SetupBrowserDxe/IfrParse.c

```diff
@@ -416,21 +440,39 @@ CreateStorage (
     ASSERT (StorageName != NULL);
 
     UnicodeString = AllocateZeroPool (AsciiStrSize (StorageName) * 2);

```

Comment:
This is another function where a `goto` could be used to free the various allocations that may have succeeded.

Reply:
Added